bucket/covergroup.py

```diff
@@ -78,7 +79,7 @@ def _set_full_path(self):
             cg._set_full_path()
 
     @validate_call
-    def include_by_function(self, matcher: Callable[[CoverBase], bool]):
+    def include_by_function(self, matcher: Callable[[CoverBase], bool], cp_only=False):
```

Comment:
I don't think it makes sense to add this argument when you can just check in the matcher function with isinstance

Reply:
Good point. Didn't think of that